### PR TITLE
complete styles

### PR DESCRIPTION
**Pull request by TracyMuso**
- Completed styling the ui

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -112,6 +112,20 @@ hr {
   color: #e8e8e8;
 }
```
Comment (Ismailco) [inline src/App.css:114]: - [ ] Kindly make sure your design looks identical to the provided Zeplin design:
  - [ ] You can center the add book form and give it a width so it's aligned with the container of the book cards
  - [ ] Also make sure the navigation is centered with the logo and links aligned left
  - [ ] Your design needs to include the profile icon on the navigation aligned to the right
  - [ ] Kindly consider sticking to the original colors, the progress circle needs to be in blue
- [ ] Kindly consider making all the suggested changes so your design replicate the original one

|Your design|Zeplin design|
|-|-|
|Screenshot_20221001_142310|image|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @TracyMuso, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TracyMuso** [comment]
I made some of the requested changes, I could not find the image with a blue progress circle, but I think that is not enough a reason to prevent the approval of this project. I made the style similar to the given project

File: src/components/Nav.js [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
-      <h1 className="logo"><Link to="/">Bookstore CMS</Link></h1>
+      <h1 className="logo"><Link style={style} to="/">Bookstore CMS</Link></h1>
       <ul className="page-links">
-        <li><Link to="/">Books</Link></li>
-        <li><Link to="Categories">Categories</Link></li>
+        <li><Link style={linkStyle} to="/">Books</Link></li>
+        <li><Link style={linkStyle} to="Categories">Categories</Link></li>
       </ul>
     </nav>
+    <img src={icon} className="icon" alt="user icon" />
```
Comment (fobadara) [inline src/components/Nav.js:24]: [REQUIRED] Kindly ensure that the icon's border is round just like it is in the template

File: src/components/Book.js [hunk 3]
```diff
@@ -28,11 +29,12 @@ function Book(props) {
               <li className="edit">Edit</li>
             </ul>
           </div>
+          <div className="circle">
+            <img src={meterTwo} alt="meter" className="progress-circle" />
+          </div>
           <div className="progress">
```
Comment (fobadara) [inline src/components/Book.js:35]: [REQUIRED] The previous reviewer asked you to ensure that the colour of the circle matches. the template which is blue. But yours is still purple.

Comment (fobadara) [inline src/components/Book.js:35]: [OPTIONAL] I noticed that you have to refresh the page before the change made from adding or removing a book is shown. I think you should refactor your code to automatically update the book list

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRS CHANGES ♻️ 
Hi @TracyMuso, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
[OPTIONAL] Kindly check out this [repo](https://github.com/fobadara/gcxchange/blob/development/README.md) for hints on how you can make your README more descriptive

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Nav.js (hunk 2 above)
Comment (TracyMuso) [reply src/components/Nav.js:24]: @fobadara , The icon from the requirements is NOT downloadable and neither is the progress bar. I changed the bar but I am not changing the icon. If you can show me where to get that exact icon they used, by all means please do.

**Comment by TracyMuso** [comment]
I have changed the progress bar
I cannot find a duplicate icon, either it has a square border or none at all

**Review by cynthiainga — APPROVED** [review]
Hi @TracyMuso, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Used correct Gitflow ✔️ 
- [ ] Good commit history ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**